modules/contact/doc/content/source/constraints/NormalMortarLMMechanicalContact.md

```diff
@@ -15,5 +15,3 @@ non-linear convergence than the pointwise method.
 !syntax inputs /Constraints/NormalMortarLMMechanicalContact
 
 !syntax children /Constraints/NormalMortarLMMechanicalContact
-
```

Comment:
What's the reason for removing this?  Is it just that there are no citations in this particular page?  I think that's just part of our standard template so that you don't need to go add that when you add a reference. I kind of think it's better to leave these in there even if they aren't used, but I guess it doesn't matter much.

Reply:
Yea, I removed them simply because they do not have actual references listed. Some documentation pages probably will not have references for some time so I suppose it would be better to add the section later when references are added.